mlir/lib/Dialect/QCO/Transforms/NativeSynthesis/PassTwoQubitWindows.cpp

```
@@ -0,0 +1,379 @@
+    emitGPhaseIfNonTrivial(rewriter, firstOp->getLoc(),
+                           best.payload.sequence.globalPhase);
+  }
+  rewriter.replaceAllUsesWith(outA, newA);
+  rewriter.replaceAllUsesWith(outB, newB);
+  for (auto* toErase : llvm::reverse(block.ops)) {
+    rewriter.eraseOp(toErase);
+  }
+  return success();
+}
+
+void collectUnitaryOpsInPreOrder(Operation* root,
```

Cppcheck (2.20.0)

[style] The function 'collectUnitaryOpsInPreOrder' is never used.

(unusedFunction)